README.md

```diff
@@ -106,9 +106,11 @@ DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
 GANDI_API_KEY='api-key'
 GITHUB_ACCESS_TOKEN='github-token'
 WATCH_ORGANIZATION='github-org'
-RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
-RABBITMQ_ADMIN_USERNAME='admin'
-RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
+DEFAULT_RABBITMQ_ADMIN_USERNAME='admin'
+DEFAULT_RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_INSTANCE_RABBITMQ_HOST='rabbitmq-dev.opencraft.hosting'
```

Comment:
@bdero  Ooooo!  I didn't even know what AMQP was!  That's seriously tidy..  thank you!